### PR TITLE
add support for referencing en prefix on docs.ros.org

### PR DESCRIPTION
**Pull request by tfoote**

### CODE REVIEW

File: _plugins/rosindex_generator.rb [hunk 1]
```diff
@@ -217,7 +218,7 @@ def map_build_result_to_icon(build_result)
 
   def get_ci_data(distro, package_name, repo_name)
     ci_data = Hash.new
-    manifest_url = '/'+distro+'/api/'+package_name+'/manifest.yaml'
+    manifest_url = '/'+DEFAULT_LANGUAGE_PREFIX+'/'+distro+'/api/'+package_name+'/manifest.yaml'
```
Comment (nuclearsandwich) [inline _plugins/rosindex_generator.rb]:
```suggestion
    manifest_url = "#{DEFAULT_LANGUAGE_PREFIX}/#{distro}/api/#{package_name}/manifest.yaml"
```

File: _plugins/rosindex_generator.rb [hunk 2]
```diff
@@ -238,7 +239,7 @@ def get_ci_data(distro, package_name, repo_name)
     end
     ci_data['job_url'] = manifest_yaml['devel_jobs'][0]
     # get additional test information if available
-    results_url = '/'+distro+'/devel_jobs/'+repo_name+'/results.yaml'
+    results_url = '/'+DEFAULT_LANGUAGE_PREFIX+'/'+distro+'/devel_jobs/'+repo_name+'/results.yaml'
```
Comment (nuclearsandwich) [inline _plugins/rosindex_generator.rb]:
```suggestion
    results_url = "#{DEFAULT_LANGUAGE_PREFIX}/#{distro}/devel_jobs/#{repo_name}/results.yaml"
```

**Review by nuclearsandwich — APPROVED** [review]
Looks good although I'd suggest using interpolation for the urls.

**Comment by ros-discourse** [comment]
This pull request has been mentioned on **ROS Discourse**. There might be relevant details there:

https://discourse.ros.org/t/migration-of-docs-ros-org-content-into-en-prefix/16829/1

**Pull request merged**